pkgs/development/tools/misc/hydra/common.nix

```diff
@@ -84,7 +84,7 @@ in stdenv.mkDerivation rec {
       perlDeps perl nix
       postgresql # for running the tests
       nlohmann_json
-      boost
+      boost mdbook
```

Comment:
mdbook should be in nativeBuildInputs, as it's used to build documentation.

Reply:
cc @shlevy, I only applied the patches onto the release branch.

Reply:
@jonringer :facepalm: sorry, sloppy on my part

Reply:
Fixing now, just testing the build

Reply:
https://github.com/NixOS/nixpkgs/commit/d47ac892331be148fd4d078c37651b4e1693414e

Reply:
Applied here as well.